rules/integrations/aws/credential_access_new_terms_secretsmanager_getsecretvalue.toml

```diff
@@ -115,7 +124,7 @@ reference = "https://attack.mitre.org/tactics/TA0006/"
 
 [rule.new_terms]
 field = "new_terms_fields"
-value = ["user.id", "aws.cloudtrail.request_parameters"]
+value = ["aws.cloudtrail.user_identity.arn"]
```

Comment:
So this is identifying the first time an identity is accessing any secret value using those APIs. Not specifically accessing a particular secret value for the first time, was that the intention?